src/controllers/music/music.html

```diff
@@ -2,7 +2,7 @@
 
     <style>
         @media all and (max-width: 32em) {
-
+  
```

Comment:
It looks like there is a lot of whitespace added in this file that shouldn't be there.

Reply:
Not sure what caused that, it's been removed. Maybe Webstorm was feeling saucy